### ISSUE TITLE
`disconnectWalletFromDapp()` and `disconnectWalletFromAllDapps()` don't work

### ISSUE THREAD

**Issue by fionnachan** [Synthetixio/synpress#510]
I'm running 2 spec files and in each of them included the following before the tests
```typescript
before(() => {
  cy.disconnectMetamaskWalletFromAllDapps()
})
```

Metamask could not be disconnected from the dApp before running the second spec file.

I manually disconnected it and the tests worked normally.

![image](https://user-images.githubusercontent.com/13184582/191230170-68d00dcf-bb3a-45ef-84f6-21ccd395eadc.png)

Using 2.3.2

**Comment by drptbl** [comment]
@fionnachan Thanks for reporting ❤️. This has been fixed in `@synthetixio/synpress@2.3.3-beta.0`.

**Comment by fionnachan** [comment]
Amazing! Thank you!!
